nvflare/fuel/hci/chunk.py

```diff
@@ -0,0 +1,209 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import struct
+
+from .checksum import Checksum
+
+HEADER_STRUCT = struct.Struct(">BII")  # marker(1), seq_num(4), size(4)
+HEADER_LEN = HEADER_STRUCT.size
+
+MARKER_DATA = 101
+MARKER_END = 102
+
+MAX_CHUNK_SIZE = 1024 * 1024
+
+
+def get_slice(buf, start: int, length: int):
+    return buf[start : start + length]
```

Comment:
Use memoryview here could improve performance a little.

Reply:
I think it's like a 'preference' or 'cosmetic issue.'